sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -3,6 +3,11 @@
 ## 1.5.0-beta.1 (Unreleased)
```

Comment:
Do we want to keep this as beta or release a new stable version?

Reply:
We want a new stable version, so should I update it to 1.5.0 ?